packages/react-core/src/helpers/util.ts

```diff
@@ -262,32 +262,84 @@ export const setBreakpointCssVars = (
     {}
   );
 
+export interface Mods {
+  default?: string;
+  sm?: string;
+  md?: string;
+  lg?: string;
+  xl?: string;
+  '2xl'?: string;
+  '3xl'?: string;
+}
+
 /**
  * This function is a helper for turning arrays of breakpointMod objects for data toolbar and flex into classes
  *
  * @param {object} mods The modifiers object
  * @param {any} styles The appropriate styles object for the component
  */
 export const formatBreakpointMods = (
-  mods: {
-    default?: string;
-    sm?: string;
-    md?: string;
-    lg?: string;
-    xl?: string;
-    '2xl'?: string;
-    '3xl'?: string;
-  },
+  mods: Mods,
   styles: any,
-  stylePrefix: string = ''
-) =>
-  Object.entries(mods || {})
+  stylePrefix: string = '',
+  breakpoint?: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | '2xl'
```

Comment:
My only feedback, I'd like to see `xs` -> `default` until we have an actual value for `xs`.